python/ray/tests/test_ray_debugger.py

```diff
@@ -77,7 +76,12 @@ def f():
 
     # Make sure that calling "continue" in the debugger
     # gives back control to the debugger loop:
-    p = pexpect.spawn("ray debug")
+    if sys.platform == "win32":
+        from pexpect.popen_spawn import PopenSpawn
+
+        p = PopenSpawn("ray debug", encoding="utf-8")
+    else:
+        p = pexpect.spawn("ray debug")
```

Comment:
pexpect.spawn does not work on windows, pexpect.PopenSpawn does work on windows.